### PR TITLE
virsh.boot: Adaption to qemu-kvm update

### PR DESCRIPTION
**Pull request by hahakiki2010**
Install qemu-kvm according to the os version. For RHEL-7, install qemu-kvm-rhev,
otherwise install qemu-kvm

Signed-off-by: Yan Li <yannli@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/bios/virsh_boot.py [hunk 1]
```diff
@@ -217,6 +217,13 @@ def setup_test_env(params, test):
     global cleanup_iso_file
     global cleanup_image_file
 
+    os_version = params.get("os_version")
```
Comment (kylazhang) [inline libvirt/tests/src/bios/virsh_boot.py]: I think we can get it from current os

Comment (hahakiki2010) [reply libvirt/tests/src/bios/virsh_boot.py]: Yes, it's better. But current method shall work enough as the version is dynamically got from CI run.

File: libvirt/tests/cfg/bios/virsh_boot.cfg [hunk 2]
```diff
@@ -13,6 +13,8 @@
     status_error = "no"
     disk_format = "qcow2"
     driver_type = "qcow2"
+    # In format of 'RHEL-7.5'
```
Comment (chunfuwen) [inline libvirt/tests/cfg/bios/virsh_boot.cfg]: I suggest remove specific RHEL information here

Comment (hahakiki2010) [reply libvirt/tests/cfg/bios/virsh_boot.cfg]: removed

File: libvirt/tests/src/bios/virsh_boot.py [hunk 3]
```diff
@@ -225,8 +229,11 @@ def setup_test_env(params, test):
         if not utils_package.package_install('OVMF'):
             test.error("OVMF package install failed")
 
-        if not utils_package.package_install('qemu-kvm-rhev'):
+        if os_version == "RHEL-7" and not \
+                utils_package.package_install('qemu-kvm-rhev'):
```
Comment (chunfuwen) [inline libvirt/tests/src/bios/virsh_boot.py]: suggest just check whether any qemu-kvm-rhev and qemu-kvm exist , if not , just skip this case. 
No need do additional work to install them.

Comment (hahakiki2010) [reply libvirt/tests/src/bios/virsh_boot.py]: change to check whether qemu-kvm is installed, if not raise test error instead.  Here still keep test error as previous script do. And I think this is part of test env preparation, so raise test error  is also reasonable.

**Pull request merged**